instance/models/mixins/openedx_monitoring.py

```diff
@@ -198,6 +223,26 @@ def delete(self, *args, **kwargs):  # pylint: disable=arguments-differ
         super().delete(*args, **kwargs)
 
 
+class NewRelicSSLMonitor(models.Model):
+    """
+    A New Relic Synthetics ssl monitor for an instance.
+    """
+    id = models.CharField(max_length=256, primary_key=True)
```

Comment:
Just out of curiosity, why do we need primary key as `CharField` here?

Reply:
Oh because we are using NewRelic generated ids which are string, right?

Reply:
Yes, I based it on the existing NewRelicAvailabilityMonitor.